internals/notifier.py

```diff
@@ -165,26 +172,27 @@ def make_email_tasks(feature, is_update=False, changes=[]):
     'You are CC\'d on this feature'
   )
   accumulate_reasons(
-      addr_reasons, feature_watchers,
+      addr_reasons, watcher_emails,
       'You are watching all feature changes')
 
   # There will always be at least one component.
   for component_name in feature.blink_components:
-    component = user_models.BlinkComponent.get_by_name(component_name)
+    component = BlinkComponent.get_by_name(component_name)
     if not component:
       logging.warning('Blink component "%s" not found.'
                       'Not sending email to subscribers' % component_name)
       continue
-
+    owner_emails: list[str] = [owner.email for owner in component.owners]
```

Comment:
The type annotations on local variables feel a little cumbersome here.  But, I suppose that mypy cannot infer the type of NDB model fields.

Reply:
The issue is that each field can optionally be repeated, so a StringProperty for example has a return type of either `None`, `str`, or `list[str]`. That being said, these type hints should be mostly optional, so it hopefully won't get in the way of development. If it's too cumbersome, we can look at ways to mitigate it.